### PR TITLE
remove duplicate variables and make variables more consistent

### PR DESCRIPTION
**Pull request by rhiGriff**
# Motivation and Context
Make environment variables consistent across all services

# What has changed

# How to test?

# Links

### CODE REVIEW

**Review by insacuri — APPROVED** [review]
LGTM. Gonna approve as flying the pipeline will show up quickly enough if it's wrong

**Review by AndrewTorrance — APPROVED** [review]

File: pipelines/performance.yml [hunk 1]
```diff
@@ -90,47 +88,23 @@ templates:
...
-      RAS_SECURE_MESSAGING_SERVICE_PORT: '80'
+      SAMPLE_SERVICE_HOST: rm-sample-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      SAMPLE_SERVICE_PORT: '80'
+      OAUTH_SERVICE_HOST: ras-rm-auth-service-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      OAUTH_SERVICE_PORT: '80'
+      FRONTSTAGE_HOST: ras-frontstage-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      FRONTSTAGE_PORT: '80'
+      SECURE_MESSAGING_SERVICE_HOST: ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
+      SECURE_MESSAGING_SERVICE_PORT: '80'
       SECURE_MESSAGE_URL: http://ras-secure-message-((performance_cloudfoundry_space)).((performance_cloudfoundry_apps_domain))
```
Comment (mfcrocker) [inline pipelines/performance.yml:99]: This one is inconsistent with the 2 above

**Review by mfcrocker — DISMISSED** [review]
One line to sort out

File: pipelines/performance.yml (hunk 1 above)
Comment (rhiGriff) [reply pipelines/performance.yml:99]: There are 2 there's secure message and secure messaging I left them both as I didn't know which one is needed

**Pull request merged**